src/pydolphinscheduler/core/task.py

```diff
@@ -169,6 +195,18 @@ def __init__(
             self.workflow = kwargs.pop("process_definition")
         else:
             self.workflow: Workflow = workflow or WorkflowContext.get()
+
+        if "local_params" in kwargs:
```

Comment:
also for L226, and when we remove both of them, we should add attr check in function `local_params`, because `self._local_params` only assign in L206 when `local_params` in `kwargs`, means it is optional, so we should check wether it exists or not first

```diff
- local_params = copy.deepcopy(self._local_params)
+ local_params = copy.deepcopy(self._local_params) if hasattr(self, "_local_params") or []
```